Cargo.toml

```diff
@@ -16,8 +16,8 @@ enable_unstable_features_that_may_break_with_minor_version_bumps = []
 
 [dependencies]
 base64 = "0.10.1"
-humantime = "2.0.0"
 indexmap = "1.0.2"
+chrono = "0.4.10"
```

Comment:
Can we disable as many optional features as possible. I used chrono initially but replaced it because of its large dependency tree.